docs/flashing.md

```diff
@@ -119,6 +119,31 @@ Flashing sequence:
 3. Flash a .hex file
 4. Reset the device into application mode (may be done automatically)
 
+## USBasploader
+
+USBasploader is a bootloader developed by matrixstorm. It is used in some atmega328p or similar devices which only function using the VUSB.
+
+To ensure compatibility with the USBasploader bootloader, make sure this block is present your `rules.mk`:
+
+    # Bootloader
```

Comment:
This block should be triple-backticked, so it can be syntax highlighted.

Reply:
I followed what the other sections followed. which is not triple-backtick

Reply:
@fauxpark I think this can be put off to later. I want to do a big overhaul of the flashing docs (not just this file), once both this, and my `:flash` target PR get merged.